.gitignore

```diff
@@ -0,0 +1 @@
+Test.md
```

Comment:
**Optional Changes**
- [ ] This file should be named `test.md` not `Test.md`. It is important to be kin on details as it implies that you are attentive to details.